gradio/themes/base.py

```diff
@@ -93,6 +94,30 @@ def repl_func(match):
 
         return f"{css_code}\n{dark_css_code}"
 
+    def _get_computed_value(self, property: str, depth=0) -> str:
+        MAX_DEPTH = 100
+        if depth > MAX_DEPTH:
```

Comment:
don't want to raise an exception in case there were "broken" themes with circular references before. They didn't error out before and don't want a breaking change.